sqlmesh/migrations/v0011_add_model_kind_name.py

```diff
@@ -55,7 +55,7 @@ def migrate(state_sync):  # type: ignore
                 "identifier": exp.DataType.build(text_type),
                 "version": exp.DataType.build(text_type),
                 "snapshot": exp.DataType.build("text"),
```

Comment:
This is missing the `text_type`.

Reply:
No, this can't be a text type. Text type is only used for text columns that can be part of a key / index. It has a limited number of characters and shouldn't be applied to `snapshot`.

Reply:
I believe the max fixed length for a key/index is higher in MSSQL than in MySQL - should I update it to use that?

And we're setting it for `kind_name` solely due to type (not length). Should I max out the length for kind_name, or reuse the key/index length?